app/(app)/parking-cards/verification/index.tsx

```diff
@@ -12,57 +13,77 @@ import Panel from '@/components/shared/Panel'
 import Typography from '@/components/shared/Typography'
 import { useTranslation } from '@/hooks/useTranslation'
 import { clientApi } from '@/modules/backend/client-api'
-import { VerifyEmailsDto } from '@/modules/backend/openapi-generated'
-
-/*
- * TODO
- * - [ ] add email validation
- * - [ ] handle error on mutation (email not sent)
- */
+import { SERVICEERROR, VerifyEmailsDto } from '@/modules/backend/openapi-generated'
+import { isServiceError } from '@/utils/errorService'
+import { isValidEmail } from '@/utils/isValidEmail'
 
 const Page = () => {
   const t = useTranslation('AddParkingCards')
 
   const [email, setEmail] = useState('')
 
-  // TODO add email validation
-  const isValid = email.includes('@')
+  const [expectedError, setExpectedError] = useState<string | null>(null)
```

Comment:
Try if this works too: `useState<string>(null)` (without `| null`, I think it should)